### ISSUE TITLE
Disabling concealing for anything that changes indentation

### ISSUE THREAD

**Issue by ivanperez-keera** [enomsg/vim-haskellConcealPlus#13]
Hi

I'm new to conceal. I've started using it because I'm using haskell-vim-now.

While I find it nice for things such as arrows and lambdas, I think layout is very informative and a touchy subject for programmers.

I think it would be great if hcp let us disable everything that does not fully respect the layout (meaning that the concealed string and the concealer require the same amount of characters).

What do you think?

**Comment by oblitum** [comment]
I also agree this is a pesky issue. I use hcp for post code online solely b/c I unable to get around using it with my cursor position jumping around and also strange indentation. I was discussing this in the purescript channel and was speculating about having an option to fill with extra whitespace characters surrounding a single character that conceals many, to match the number of characters concealed and avoid this problem. Of course distribution of such extra whitespace around the special character is another visual issue. Not sure whether it would work though.
